src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -140,6 +143,7 @@ class Comment extends React.Component<Props, State> {
                                 <Media.Menu
                                     isDisabled={isConfirmingDelete}
                                     data-testid="comment-actions-menu"
+                                    aria-label={commentOptionsAriaLabel}
```

Comment:
This is minor, but we try to alphabetize component props, object properties, variable declarations, etc. when possible.